Swent-team-6/icebreakrr
# update end to end 1&2

This PR starts the final end-to-end tests. I tweaked the two tests that where already made so that one test is the user flow of the Around You and the other one is the user flow of the Settings

Around you flow : 
- check that the filter works
- check that you can click on profiles
- tries to send a request and cancel it (this will certainly be moved in another end-to-end test next week)

Settings flow : 
- test the profile edition
- test the toggle discoverability
- check the block functionality
- check the report functionality
- check the already met functionnality 
- log out

I also added a way to cancel a meeting request you have sent in the notifications.


To be done next week : 
- Make a end-to-end test for the notification page and send messages (moved from Around you flow)
- Make a end-to-end test for when someone accepts your request
